target/linux/mediatek/image/mt7622.mk

```diff
@@ -203,6 +203,16 @@ define Device/mediatek_mt7622-rfb1-ubi
 endef
 TARGET_DEVICES += mediatek_mt7622-rfb1-ubi
 
+define Device/mediatek_mt7622-rfb-mtk0897-ubi
+  DEVICE_VENDOR := MediaTek
+  DEVICE_MODEL := MTK7622 MTK0897 UBI
+  DEVICE_DTS := mt7622-rfb-mtk0897-ubi
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_PACKAGES := swconfig
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
```

Comment:
As you have enabled `CONFIG_FIT_PARTITION` you could use a single uImage.FIT instead of sysupgrade.tar. See BPi-R64 and E8450/RT3200 for examples.

Reply:
I know about that one and I applied it to the mt7629 board.
I flashed this MT7622 reference board I have with the old u-boot-2014 and enabled ubi support in this ancient u-boot. (UBI on top of MTK BMT, just for testing.) The new FIT image doesn't work with the old u-boot.
BTW Neither device support works with the original bootloaders and I don't plan to push the two device support patches to master. That's why they don't have proper commit messages. Sorry for wasted reviewing efforts :D

Reply:
Yes, you need U-Boot 2018.01 or later for external-data FIT.